pkg/api.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import (
+	"io/ioutil"
+	"net/http"
+)
+
+const baseURL = "https://api.ipify.org?format=json"
```

Comment:
I do love a baseURL variable when making API calls, and I think that's a good best practice, but considering there is a single endpoint here, perhaps we should just have it inside the function in a variable called url.

Reply:
Although, you are totally right, Let's stick with the baseURL approach for now. We'll do a code clean up once the tutorial gets closer to completion.